From: Martin Michlmayr <tbm@cyrius.com>
Date: Mon, 7 Jun 2010 19:31:01 +0100
Subject: [PATCH] mtd: m25p80: Add support for Macronix 25L8005

commit 6175f4a194653fad46218167fac05386972d995c upstream.

Add support for Macronix 25L8005.  Tested on a HP t5325 Thin Client.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>
Signed-off-by: Artem Bityutskiy <Artem.Bityutskiy@nokia.com>
Signed-off-by: David Woodhouse <David.Woodhouse@intel.com>
[backported to 2.6.32 -- tbm]
---
--- a/drivers/mtd/devices/m25p80.c	2010-06-19 10:37:08.000000000 +0000
+++ b/drivers/mtd/devices/m25p80.c	2010-06-19 10:37:54.000000000 +0000
@@ -621,6 +621,7 @@
 	{ "at26df321",  0x1f4701, 0, 64 * 1024, 64, SECT_4K, },
 
 	/* Macronix */
+	{ "mx25l8005", 0xc22014, 0, 64 * 1024,  16, },
 	{ "mx25l3205d", 0xc22016, 0, 64 * 1024, 64, },
 	{ "mx25l6405d", 0xc22017, 0, 64 * 1024, 128, },
 	{ "mx25l12805d", 0xc22018, 0, 64 * 1024, 256, },

